dropbox/1.0/modules/ROOT/pages/dropbox-connector-reference.adoc

```diff
@@ -0,0 +1,9718 @@
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  How the connector streams request content to the remote service:
+
+* `AUTO` (default)
+
+Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`
+
+Connector always streams the request content.
+
+* `NEVER`
+
+Connector does not stream the request content. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Any
+| Attributes Type a| <<HttpResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>>
+
+=== Throws
+
+* DROPBOX-USERV2:BAD_REQUEST
+* DROPBOX-USERV2:CLIENT_ERROR
+* DROPBOX-USERV2:CONNECTIVITY
+* DROPBOX-USERV2:INTERNAL_SERVER_ERROR
+* DROPBOX-USERV2:NOT_ACCEPTABLE
+* DROPBOX-USERV2:NOT_FOUND
+* DROPBOX-USERV2:RETRY_EXHAUSTED
+* DROPBOX-USERV2:SERVER_ERROR
+* DROPBOX-USERV2:SERVICE_UNAVAILABLE
+* DROPBOX-USERV2:TIMEOUT
+* DROPBOX-USERV2:TOO_MANY_REQUESTS
+* DROPBOX-USERV2:UNAUTHORIZED
+* DROPBOX-USERV2:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateSharingListFoldersContinue]]
+== Sharing List Folders Continue
+`<dropbox-userv2:create-sharing-list-folders-continue>`
+
+
+Once a cursor has been retrieved from `list_folders`, use this to paginate through all the shared folders. The cursor must come from a previous call to `list_folders` or `list_folders/continue`. Apps must have full Dropbox access to use this endpoint. This operation makes an HTTP POST request to the `/sharing/list_folders/continue` endpoint.
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | Name of the configuration to use. | | x
+| Body a| Any |  Content to use. |  #[payload] |
+| Config Ref a| ConfigurationProvider |  Name of the configuration to use to execute this component. |  | x
+| Streaming Strategy a| * <<RepeatableInMemoryStream>>
+* <<RepeatableFileStoreStream>>
+* non-repeatable-stream |  Configures how Mule processes streams. Repeatable streams are the default behavior. |  |
+| Custom Query Parameters a| Object | Custom query parameters to include in the request. The specified query parameters are merged with the default query parameters that are specified in the configuration. |  |
+| Custom Headers a| Object | Custom headers to include in the request. The specified custom headers are merged with the default headers that are specified in the configuration. |  |
+| Response Timeout a| Number |  How long Mule waits for a response to complete before the response times out. |  |
+| Response Timeout Unit a| Enumeration, one of:
+
+** N
```

Comment:
```suggestion
Deletes all of the access token information of a given resource owner ID so that it is impossible to execute any operation for that user without repeating the authorization dance.
```